DiamondLightSource/python-artemis
# #481 Add beamline field to graylog

Fixes #481

### To test:
1. Run: 
```
source .venv/bin/activate
export BEAMLINE=i04-1
python -m artemis
```
1. Check the logs in https://graylog2.diamond.ac.uk/streams/631745613d8af00a49bfee55/search and confirm the latest log is from you and contains the expected beamline
